modules/cloud-management-platforms-paladin-cloud.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-cloud-sources.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="cloud-management-platforms-paladin-cloud_{context}"]
+= Configuring Paladin Cloud integration
+
+To discover cluster assets from Paladin Cloud, create a new integration in {product-title}.
+
+.Prerequisites
+* A Paladin Cloud account.
+* A Paladin Cloud API token.
+
+.Procedure
+. On the {product-title-short} portal, navigate to *Platform Configuration* -> *Integrations*.
+. Scroll down to the *Cloud source integrations* section and select *Paladin Cloud*.
+. Click *New integration*.
+. Enter a name for *Integration name*.
+. Enter the Paladin Cloud API endpoint for *Paladin Cloud endpoint*. The default is `\https://api.paladincloud.io`.
```

Comment:
```suggestion
. Enter the Paladin Cloud API endpoint for *Paladin Cloud endpoint*. The default is `https://api.paladincloud.io`.
```
Is the backtick intentional here?

Reply:
Thanks @stehessel Yes, so that it shows as plain text in the docs and not a clickable link.